### DISCUSSION TITLE
README

### DISCUSSION THREAD

**Discussion by lunaleaps** [reactwg/react-native-releases#2] [Announcements]
# Purpose of this Discussion Repo
* Sharing status of a new minor release under the [`Releases`][releases-category] category
* Discussion of releasing a patch on the current stable release under the [`Patches`][patches-category] category

## FAQ
* [Release Schedule](https://github.com/facebook/react-native/wiki/Release-Schedule)
* [How do React Native releases work?][release-wiki]
* I've just upgraded my react-native version and have encountered a build or a fundamental workflow regression:
    * [File a release issue here][release-regression-issue-form]
* [Who is involved in a release?][release-wiki-roles]
* [How can I get involved or help with React Native releases?][release-wiki-get-involved]

[terminology]: https://github.com/facebook/react-native/wiki/Releases
[release-regression-issue-form]: https://github.com/facebook/react-native/issues/new?assignees=&labels=Needs%3A+Triage+%3Amag%3A%2CType%3A+Upgrade+Issue&template=upgrade-regression-form.yml
[releases-category]: https://github.com/reactwg/react-native-releases/discussions/categories/releases
[patches-category]: https://github.com/reactwg/react-native-releases/discussions/categories/patches 
[release-issues]: https://github.com/facebook/react-native/issues?q=is%3Aopen+is%3Aissue+label%3Apre-release
[stable-regression-form]: https://github.com/facebook/react-native/issues/new?assignees=&labels=Needs%3A+Triage+%3Amag%3A%2CType%3A+Upgrade+Issue&template=upgrade-regression-form.yml
[stable-regression-issues]: https://github.com/facebook/react-native/labels/Type%3A%20Upgrade%20Issue
[patches-category]: https://github.com/reactwg/react-native-releases/discussions/categories/patches 
[release-wiki]: https://github.com/facebook/react-native/wiki/Releases
[release-wiki-roles]: https://github.com/facebook/react-native/wiki/Release-Roles
[release-wiki-get-involved]: https://github.com/facebook/react-native/wiki/Help-with-Releases

**Comment by DEVfancybear** [comment]
We are waiting for new architecture of react native as per https://reactnative.dev/blog/2021/08/19/h2-2021.
